### ISSUE TITLE
2018.406.0 crash at startup

### ISSUE THREAD

**Issue by Trigger-b0y** [ppy/osu#2368]
v2018.406.0 crashing at startup
Previous versions works normal
I tried to reinstall game, but still not work
[performance.log](https://github.com/ppy/osu/files/1887683/performance.log)
[runtime.log](https://github.com/ppy/osu/files/1887684/runtime.log)

**Comment by FreezyLemon** [comment]
From the runtime.log:

```
2018.04.08 16:59:24: System.MissingMethodException: Метод не найден: "System.Collections.Generic.IEnumerable`1<!!0> System.Linq.Enumerable.Prepend(System.Collections.Generic.IEnumerable`1<!!0>, !!0)".
2018.04.08 16:59:24: в osu.Game.Beatmaps.Formats.Decoder`1.Decode(StreamReader primaryStream, StreamReader[] otherStreams)
2018.04.08 16:59:24: в osu.Game.Beatmaps.BeatmapManager.CreateModel(ArchiveReader reader)
2018.04.08 16:59:24: в osu.Game.Database.ArchiveModelManager`2.Import(ArchiveReader archive)
2018.04.08 16:59:24: в osu.Game.Screens.Menu.Intro.load(AudioManager audio, OsuConfigManager config, BeatmapManager beatmaps, Game game)
```

Not quite sure how this happened. It might be something related to running osu.Desktop in .NET 4.6.1 while  `Enumerable.Prepend` is present starting in .NET 4.7.. ([Source](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.prepend?view=netframework-4.7.1#System_Linq_Enumerable_Prepend__1_System_Collections_Generic_IEnumerable___0____0_))

**Comment by Herobrine59** [comment]
I've also got that error since i've downloaded the file.
I've tried to open the client where it locates at
USER > AppData > Local > osulazer >app-2018.406.0
and it does not work

**Comment by smoogipoo** [comment]
I think we'd be safe bumping osu! to net471. The only compatibility we lose is windows 8 (not 8.1) and windows 10 (vanilla - non-creators/anniversary). Thoughts, @peppy ?

**Comment by Trigger-b0y** [comment]
@smoogipoo win10 pro x64 ver.1703 build15063.786 (creators update)

**Comment by xWTyrone** [comment]
> while Enumerable.Prepend is present starting in .NET 4.7..

True. But it is also present inside .NET Core since version 1.0.

VS nagged me about it when I first cloned the repo and opened the sln file. Turned out .NET Core 2.0 had to be downloaded separately. Install that and there seems to be no problem.

(I have no idea how to test my theory tho. Here's .NET Core 2.0 DL link for future reference:)

https://www.microsoft.com/net/download/dotnet-core/runtime-2.0.7
